src/main/java/traincraft/blocks/assemblytables/TileAssemblyTableII.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Traincraft
+ * Copyright (c) 2011-2020.
+ *
+ * This file ("null.java") is part of the Traincraft mod for Minecraft.
+ * It is created by all people that are listed with @author below.
+ * It is distributed under LGPL-v3.0.
+ * You can find the source code at https://github.com/Traincraft/Traincraft
+ */
+package traincraft.blocks.assemblytables;
+
+import net.minecraft.tileentity.TileEntityLockable;
```

Comment:
why this import

Reply:
from when I was testing things, removed now